sys/include/net/gcoap/fileserver.h

```diff
@@ -90,6 +90,51 @@ extern "C" {
  */
 #define COAPFILESERVER_DIR_DELETE_ETAG (0x6ce88b56u)
 
+/**
+ * @brief   GCoAP fileserver event types
+ *
+ * @note This requires the gcoap_fileserver_callback module.
+ */
+typedef enum {
+    GCOAP_FILESERVER_GET_START,     /**< file download started   */
```

Comment:
maybe rename the enums in the manner of  `GCOAP_FILESERVER_GET_FILE_START`

Reply:
Ok